src/Directory.Build.targets

```diff
@@ -15,4 +15,8 @@
     <DefineConstants Condition=" $(IsOSXBuild) == 'true' ">$(DefineConstants);OSX_BUILD</DefineConstants>
     <DefineConstants Condition=" $(IsWindowsBuild) == 'true' ">$(DefineConstants);WINDOWS_BUILD</DefineConstants>
   </PropertyGroup>
+  
+  <ItemGroup>
+    <AdditionalFiles Include="$(MSBuildThisFileDirectory)/BannedSymbols.txt" />
```

Comment:
>AdditionalFiles [](start = 5, length = 15)

This doesn't get included in compiled binaries, right?

Reply:
No, it doesn't.